docs/source/getting-started/first-feature.rst

```diff
@@ -175,24 +179,22 @@ You can see we simply follow the formatting of the previous panels and include t
 That is all for the implementation!
 Let's give the new functionality a spin.
 
-Running the New App
+Running the New Feature
 ===================
 
-To start up the new app, open up your terminal application.
-First make sure your previously run commands are terminated, in particular the ``daml-start.sh`` command.
-You can do this by hitting ``Ctrl-C`` in the terminal window where you ran the command.
-This shuts down the previous instances of the sandbox and JSON API server: it is important that we start our new app with new instances of these components.
+As previously mentioned, we need to restart the DAML Sandbox and the JSON API, in order to get the messaging feature up and running. 
```

Comment:
I think this is misleading. This is not about restarting sandbox or the JSON API. This is about building the DAR and uploading it. It just happens that for simplicity we do not upload the DAR to the running sandbox but kill and restart it to also clear out data.

Reply:
I see your point, but it is too detailed for the getting started guide, as the effect of the action are the same. If I look in our docs this is how _daml start_ (used in the ./daml-start.sh) is described 

"Launch Sandbox, Navigator and the HTTP JSON API Service: daml start You can disable the HTTP JSON API by passing --json-api-port none to daml start. To specify additional options for sandbox/navigator/the HTTP JSON API you can use --sandbox-option=opt, --navigator-option=opt and --json-api-option=opt."

Nothing about the actual mechanics of building and uploading the DAR. Overall, think including this would make a very lengthy explanation. Once we do have the possibility to upload the DAR file to a running sandbox we can change it in here as well.

Reply:
Can we maybe make it less precise, e.g., something like “we need to run `daml start` again to restart the components with our new model”? I fully agree that a lengthy explanation is not suitable here. What I find confusing is that the current explanation is very precise in what you do (restart sandbox and the JSON API) but in an incorrect way (restarting those components on their own doesn’t do what you want here).

Reply:
Will mark this comment as resolved and will commit a new version with different text